recipes/pimsviewer/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "pimsviewer" %}
+{% set version = "1.0" %}
+{% set sha256 = "f78fc0d3431d4c9c19a5e20d05a429cce01fe9c645aa6da7c446b538e5a0473b" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
```

Comment:
Would it make sense to use [`noarch: python`]( https://conda-forge.org/docs/meta.html#building-noarch-packages ) here?

Reply:
Yes, I think it is possible, I overlooked this option. Thank you!